packages/core/__tests__/MobileCoreTests.ts

```diff
@@ -76,7 +85,7 @@ describe('MobileCore', () => {
     let testEvent = new Event('eventName', 'eventType', 'eventSource', {
       testDataKey: 'testDataValue'
     });
-    await MobileCore.dispatchEventWithResponseCallback(testEvent);
+    await MobileCore.dispatchEventWithResponseCallback(testEvent, 5000);
     expect(spy).toHaveBeenCalledWith(testEvent);
```

Comment:
`dispatchEventWithResponseCallback` is called with two arguments. Let's add the timeout argument as well.